torch/onnx/utils.py

```diff
@@ -275,6 +275,9 @@ def _model_to_graph(model, args, f, verbose=False, training=False,
     param_names = input_and_param_names[len(input_and_param_names) - len(params):]
     params_dict = dict(zip(param_names, params))
 
+    if do_constant_folding:
+        params_dict = torch._C._jit_pass_onnx_constant_fold(graph, params_dict)
```

Comment:
We can also check whether the opset is supported by our constant folding. Op spec may change, and the optimization may not be applicable any more.

Reply:
Yes, good point. Now checking for opset version (opset 9).